Language/Haskell/GhcMod/Logging.hs

```diff
@@ -78,6 +83,17 @@ gmLog level loc' doc = do
 
   gmlJournal (GhcModLog Nothing (Last Nothing) [(level, loc', msgDoc)])
 
+-- | Appends a collection of logs to the logging environment, with effects
+-- | if their log level specifies it should
+gmLog' :: (MonadIO m, GmLog m, GmOut m) => GhcModLog -> m ()
+gmLog' GhcModLog { gmLogMessages } = do
```

Comment:
I added some change for those points. I think it would be very useful for the users, so in that respect, the sooner the better...